management_api.go

```diff
@@ -96,6 +103,77 @@ func (e *Enforcer) GetFilteredNamedGroupingPolicy(ptype string, fieldIndex int,
 	return e.model.GetFilteredPolicy("g", ptype, fieldIndex, fieldValues...)
 }
 
+// FilteredPolicyWithMatcher gets rules based on matcher from the policy.
+func (e *Enforcer) FilteredPolicyWithMatcher(pType string, matcher string) ([][]string, error) {
```

Comment:
FilteredPolicyWithMatcher -> GetFilteredNamedPolicyWithMatcher

pType -> ptype

Reply:
it's ok.